src/VirtoCommerce.DescriptionExportImportModule.Data/VirtoCommerce.DescriptionExportImportModule.Data.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
UTF8 with BOM? :) Maybe revert?

Reply:
Now file has encoding UTF without BOM. It means the file had UTF-8 with BOM before changes.